Tests/SKCoreTests/ToolchainRegistryTests.swift

```diff
@@ -198,7 +198,13 @@ final class ToolchainRegistryTests: XCTestCase {
     XCTAssertNil(tr.default)
     XCTAssert(tr.toolchains.isEmpty)
 
-    try! ProcessEnv.setVar("SOURCEKIT_PATH", value: "/bogus:\(binPath):/bogus2")
+#if os(Windows)
+    let separator: String = ";"
+#else
+    let separator: String = ":"
+#endif
+
+    try! ProcessEnv.setVar("SOURCEKIT_PATH", value: "/bogus\(separator)\(binPath)\(separator)/bogus2")
```

Comment:
I think this would read nicer as
```suggestion
    try! ProcessEnv.setVar("SOURCEKIT_PATH", value: ["/bogus", "\(binPath)", "/bogus2"].joined(separator: separator))
```

Same below

Reply:
I definitely have to agree, thanks for the suggestion!

Reply:
Going to tweak it a bit further, I think that using `binPath.pathString` is nicer rather than the string interpolation for the string conversion.